packages/webdriver-utils/src/providers/automateProvider.js

```diff
@@ -187,10 +187,6 @@ export default class AutomateProvider extends GenericProvider {
     const resolution = await this.metaData.screenResolution();
     const orientation = (this.metaData.orientation() || automateCaps.deviceOrientation)?.toLowerCase();
 
-    // for android window size only constitutes of browser viewport, hence adding nav / status / url bar heights
```

Comment:
Can you also remove the function definition for `getHeaderFooter` function ?
And also it's associated calls at https://github.com/percy/cli/blob/master/packages/webdriver-utils/src/providers/genericProvider.js#L159 ?